src/style.css

```diff
@@ -1 +1,82 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, sans-serif;
+}
+
+.search-container {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  padding: 20px;
+}
+
+.search-input {
+  width: 95%;
+  padding-left: 40px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-weight: 300;
+  transition: border-color 0.3s;
+  box-shadow: 0 0 7 rgb(205, 205, 205);
+  font-family: inherit;
+  position: relative;
+}
+
+.big {
+  margin-top: 20px;
+  height: 70px;
+  font-size: 16px;
+}
+
+.small {
+  margin-top: 20px;
+  height: 42px;
+  font-size: 14px;
+  box-shadow: 0 0 7 rgb(217, 216, 216);
+}
+
+.search-input:hover {
+  border-color: #e1e7ed;
+  box-shadow: 0 3px 5px rgb(153, 153, 153);
+}
+
+.search-input:focus {
+  background:
+    linear-gradient(#ffff, #f6f6f7),
+    url(../src/images/Search.svg) no-repeat 30px;
+  color: #3d4e61;
+  font-weight: 600;
+  font-family: inherit;
+  outline: none;
+}
+
+.icon::before {
+  /* background: url(../src/images/Search.svg) no-repeat; */
+  content: url(../src/images/Search.svg);
```

Comment:
The content URL in the .icon::before selector should be relative to the CSS file as well. Make sure the path is correct based on the project's folder structure.